src/Framework/Microsoft.AspNetCore.App/Microsoft.AspNetCore.App.props

```diff
@@ -61,16 +51,11 @@
     <Dependency Include="Microsoft.AspNetCore.Mvc.Formatters.Json"                     Version="$(MicrosoftAspNetCoreMvcFormattersJsonPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.Formatters.Xml"                      Version="$(MicrosoftAspNetCoreMvcFormattersXmlPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.Localization"                        Version="$(MicrosoftAspNetCoreMvcLocalizationPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Mvc.Razor.Extensions"                    Version="$(MicrosoftAspNetCoreMvcRazorExtensionsPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.Razor"                               Version="$(MicrosoftAspNetCoreMvcRazorPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.RazorPages"                          Version="$(MicrosoftAspNetCoreMvcRazorPagesPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.TagHelpers"                          Version="$(MicrosoftAspNetCoreMvcTagHelpersPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc.ViewFeatures"                        Version="$(MicrosoftAspNetCoreMvcViewFeaturesPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.Mvc"                                     Version="$(MicrosoftAspNetCoreMvcPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.NodeServices"                            Version="$(MicrosoftAspNetCoreNodeServicesPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Owin"                                    Version="$(MicrosoftAspNetCoreOwinPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Razor.Language"                          Version="$(MicrosoftAspNetCoreRazorLanguagePackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Razor.Runtime"                           Version="$(MicrosoftAspNetCoreRazorRuntimePackageVersion)" />
```

Comment:
`Microsoft.AspNetCore.Mvc.Razor` references this package, primarily for runtime compilation. Do we need to do something to avoid breaking things that use the runtime (e.g. templates \ MusicStore)?

Reply:
I was under the impression that we were removing runtime compilation from the shared framework, but it was also unclear to me what those needing runtime compilation are supposed to do in 3.0.

Reply:
@rynowak

Reply:
Some of these dependencies might need to continue to exist in the shared fx  e.g. `Microsoft.AspNetCore.Razor.Runtime` has the APIs to load build time compiled Razor assemblies. While the plan is to to remove runtime compilation from MVC, we haven't done the work as yet i.e. Mvc.Razor still references these assemblies. Is it safe to remove these dependencies without doing the work in Mvc?

Reply:
This can't be removed. Without going into a ton of detail `Microsoft.AspNetCore.Razor.Runtime` contains `ITagHelper` which users reference to create TagHelpers.

Reply:
Well, it type forwards ITagHelper: https://github.com/aspnet/Razor/blob/master/src/Microsoft.AspNetCore.Razor.Runtime/Properties/AssemblyInfo.cs#L13

Reply:
I double checked my list, you're right, Razor.Runtime was on the "keep" list, but Razor.Language was on the remove list. Will update.

Reply:
Razor.Runtime's moved here - https://github.com/aspnet/AspNetCore/pull/4004/files#diff-18acda8908401dcd9ef271b4693032beR60